src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
you can make a destructuring from person
```suggestion
export const Person = ({ person }) => {
  const { ... } = person;
  
  return (
    <section className="Person">
```